java/pom.xml

```diff
@@ -321,6 +321,12 @@
       <artifactId>google-cloud-bigtable</artifactId>
       <version>2.1.0</version>
     </dependency>
+    <!-- Spark kafka Direct stream-->
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-streaming-kafka-0-10_${scala.binary.version}</artifactId>
+      <version>${spark.version}</version>
+    </dependency>
```

Comment:
This is already added in pom.xml as part of #897 
Please remove.

Reply:
done